build.gradle

```diff
@@ -0,0 +1,107 @@
+buildscript {
+  ext.kotlin_version = "[1.2,)"
+
+  dependencies {
+    repositories {
+      mavenCentral()
+      jcenter()
+    }
+    classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+    classpath "org.jetbrains.dokka:dokka-gradle-plugin:0.9.17"
+  }
+}
+
+apply plugin: "kotlin"
+apply plugin: "checkstyle"
+apply plugin: "org.jetbrains.dokka"
+apply plugin: "maven-publish"
+apply plugin: "signing"
+
+group = "software.amazon.ion"
+version = "0.8.0-SNAPSHOT"
+
+repositories { mavenCentral() }
+
+dependencies {
+  compile "software.amazon.ion:ion-java:[1.2,)"
+  compile "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
+  testCompile "org.jetbrains.kotlin:kotlin-test-junit:$kotlin_version"
+}
+sourceSets {
+  main.java.srcDirs = ["src"]
+  main.kotlin.srcDirs = ["src"]
+  test.java.srcDirs = ["test"]
+  test.kotlin.srcDirs = ["test"]
+}
+
+checkstyle {
```

Comment:
Not sure this is doing anything since there is no java here.

Reply:
removed